Dockerfile

```diff
@@ -50,7 +50,10 @@ RUN pip3 install \
 COPY --from=nodebuild /usr/local/src/app/packages ./packages
 COPY --from=nodebuild /usr/local/src/app/node_modules ./node_modules
 COPY --from=nodebuild /usr/local/src/app/package.json ./package.json
-COPY buildinfo ./buildinfo
+
+# wildcard used to act as 'copy if exists'
+COPY . buildinfo* ./buildinfo
```

Comment:
Is the `.` here on purpose?  I think that might cause it to copy the entire working directory into the docker container, which will cause the resulting image to be huge.

Reply:
THis was a mistake - cleaned up